mne/epochs.py

```diff
@@ -265,7 +265,8 @@ def __init__(self, info, data, events, event_id=None, tmin=-0.2, tmax=0.5,
 
         if events is not None:  # RtEpochs can have events=None
             if events.dtype.kind not in ['i', 'u']:
-                raise ValueError('events must be an array of type int')
+                raise ValueError('events must be an array of type int, got type'
+                                 '%s' % (events.dtype.kind))
```

Comment:
events.dtype.kind -> events.dtype

besides LGTM

Reply:
Ah yes, thanks! Actually, I just realized that `events` is cast to a np.ndarray at line 244. Should we try to catch the type before `events` is cast to an array to make the error message easier to understand?

Reply:
@agramfort just checking if you agree with my comment above ^^^